test/app-tap/gh-5040-inter-mode-isatty-via-errinj.test.lua

```diff
@@ -7,8 +7,7 @@ local fio = require('fio')
 
```

Comment:
How about `test/app-tap/gh-2717-no-quit-sigint.test.lua`?

Reply:
Thx, missed that.